docs/upgrade/troubleshooting.md

```diff
@@ -128,13 +131,54 @@ The Harvester controller deletes the upgrade repository VM and all files that ar
     hvst-upgrade-9gmg2   10m
     ```
 
-3. Delete the Upgrade CR
+1. Delete the Upgrade CR
 
     ```
     $ kubectl delete upgrade.harvesterhci.io/hvst-upgrade-9gmg2 -n harvester-system
     ```
 
-4. Click the upgrade button in the Harvester dashboard to start an upgrade again.
+1. Resume the following Managedcharts
```

Comment:
Though it does not harm the cluster running the script even when not doing upgrades, I think we can add more description about why they need to run this (actually, they don't need to if the previous upgrade did not get past a certain point, but I get your point).

Reply:
Updated, thanks.